README.md

```diff
@@ -30,10 +29,17 @@
     - [Creating a new container](#creating-a-new-container)
 
 
-<h1><a id="important">Important</a></h1> 
-To pull the image, click on the tab **Tags** and choose the correct Docker command from there.
+<h1><a id="support">Important</a></h1>
+
+> [!WARNING] 
+**WARNING**: To pull the image, click on the tab **Tags** and choose the correct Docker command from there.
 **DO NOT** attempt to pull the image from this page (Overview).
```

Comment:
Why doesn't SAP publish the image also with `latest` tag (which is common practice)?